src/services/BrandingService.js

```diff
@@ -0,0 +1,75 @@
+import Service from '../abc/Service';
+
+export default class BrandingService extends Service {
+
+	/*
+	Branding service expects `header-logo-full.svg` and `header-logo-minimized.svg` SVG images
+	in `/public/media/logo/` directory:
+
+	* `header-logo-full.svg` dimensions: 120 x 30 pixels
+	* `header-logo-minimized.svg` dimensions: 30 x 30 pixels
+	*/
+
+	constructor( app, serviceName="BrandingService" ){
+		super(app, serviceName)
+		this.state = [];
+		this.App = app;
+
+		this.brandImage = undefined;
+		this.defaultBrandImage = undefined;
+		this.sidebarLogo = undefined
+	}
+
+	initialize() {
+		// which of the if statements is 'more' true?..
+		// if (this.App.Services.ConfigService && this.App.Modules.some(obj => obj.Name == "ASABConfigModule")) {
+		if (this.App.Services.ConfigService) {
```

Comment:
Hi @Pe5h4 , I'm not sure how to approach this condition. I found the one on line 25 in [Sidebar service]()https://github.com/TeskaLabs/asab-webui/blob/master/src/containers/Sidebar/service.js#L18, when accessing title. But is it necessary? Thank you!

Reply:
@petrKavulok `this.App.Modules.some(obj => obj.Name == "ASABConfigModule"` part is not necessary in this case

Reply:
ASABConfigModule is https://github.com/TeskaLabs/asab-webui/tree/master/src/modules/maintenance/ConfigModule - which is a different part of the app you dont need here in Branding service